glossary.tex

```diff
@@ -33,6 +33,11 @@
   name={FTP},
   description={File Transfer Protocol, used to copy files between distinct machines (over a network.) FTP is unencrypted, and as such blocked on certain systems. SFTP or SCP are secure alternatives to FTP},
 }
+\newglossaryentry{GPGPU}
+{
+  name={GPGPU},
+  description={A general purpose graphical processing unit. A GPGPU is a consumable resource. A GPGPU is a GPU that performs non-specialized calculations that would typically be conducted by the CPU. A compute node may contain zero, one or more GPGPUs},
```

Comment:
The `non-specialized` is a bit unclear to me here...

Reply:
How about:  "A GPGPU is a GPU that is used for highly parallel general purpose calculations." ?

Reply:
:+1: